shell/components/dialog/ForcePodRemoveDialog.vue

```diff
@@ -0,0 +1,218 @@
+<script>
```

Comment:
There should be a way of doing this without a large amount of copy and paste from PromptRemove. It looks like you should be able to achieve something similar with via the custom prompt remove feature (see shell/promptRemove/kubevirt.io.virtualmachine.vue)